packages/beacon-node/src/sync/unknownBlock.ts

```diff
@@ -414,7 +414,8 @@ export class BlockInputSync {
         ignoreIfFinalized: true,
         blsVerifyOnMainThread: true,
         // block is validated with correct root, we want to process it as soon as possible
-        eagerPersistBlock: true,
+        // however, due to other optimizations, we don't eagerly persist the block
+        eagerPersistBlock: false,
```

Comment:
same to above, if we do this postfulu only, we don't have to worry about the current validator performance of electra